charts/index.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
   secrets-store-csi-driver:
   - apiVersion: v1
-    appVersion: 0.0.9
+    appVersion: 0.0.10
```

Comment:
The index yaml needs to contain both the v0.0.9 and v0.0.10 versions. 

You can generate it by running these steps -
```bash
cd charts
helm package secrets-store-csi-driver
helm repo index . --url https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts
```

This will generate the package and update the index

Reply:
we should add a script to automate this -  opened an issue to track it https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/201

Reply:
oops good call.